frontend/tet/admin/src/components/jobPostings/__tests__/JobPostingsListItem.test.tsx

```diff
@@ -61,10 +66,20 @@ describe('JobPostingsListItem', () => {
     await within(list).findByText(/poista/i);
   });
 
-  it('it should show title, organization name and description', async () => {
+  it('it should show title, organization name, description and dates', async () => {
     renderComponent(<JobPostingsListItem posting={published} />);
 
     await screen.findByText(new RegExp(`${published.title} - ${published.org_name}`, 'i'));
     await screen.findByText(new RegExp(`${published.description}`, 'i'));
+    await screen.findByText(new RegExp(`${published.start_date}-${published.end_date}$`, 'i'), {
+      normalizer: (str) => str.replace(/\s/g, ''),
+    });
+  });
+
+  it('it should hide end date when furthest end date special value used', async () => {
+    renderComponent(<JobPostingsListItem posting={never_expiring} />);
+    await screen.findByText(new RegExp(`${never_expiring.start_date}-$`, 'i'), {
```

Comment:
If I understand the regular expression correctly this only tests that "<start_date>-" is found in
the string but it doesn't say anything about what comes or doesn't come after it.

Reply:
Actually the `$` in the end checks that there is nothing coming after that.

Reply:
Ah, so it's testing that a string ends, next is "<start_date>-" and that string ends. Hmm... so there's a row with only "<start_date>-" in the text, right?

Reply:
Yes, the idea is to check that when end date is the special value `2999-12-31` then it does not display `<start_date>-2999-12-31` but just `<start_date>-` instead

Reply:
Ah, yes, the first $ is related to the variable evaluation i.e. `${x}` evaluates variable x but the last $ is the end of string regular expression marker, alright.

Reply:
For more clarity I changed it to `^${never_expiring.start_date}-$`

Reply:
> Left some comments, please have a look at them.
> 
> backend/tet/events/linkedevents.py: in `list_ongoing_events_authenticated` function this could be also removed if doing changes to the pull request:
> 
> ```python
> # TODO exclude ended events
> # "start": "2021-01-01",
> # "end"
> ```
> 
> as the expired events are not excluded but partitioned into a different group afterwards now.

Removed